### DISCUSSION TITLE
Typescript mouse event inside some parts of code

### DISCUSSION THREAD

**Discussion by HalilFocic** [maxGraph/maxGraph#564] [Development]
Hello. I've noticed that inside GraphView file the isScrollEvent function takes MouseEvent but if we are on iPad i actually receive TouchEvent. I think function param type as well as body should change. 

Basically we need to read clientX and Y inside touches array instead of accessing it directly.

I've done some changes locally and error is not showing up and its working much better on my iPad. I would love to discuss more before making a PR since I would love to contribute to issues on tablets.

**Comment by tbouffard** [comment]
This discussion continued in #562, a PR has been proposed to fix this: see #566
